Formula/stanford-corenlp.rb

```diff
@@ -1,9 +1,9 @@
 class StanfordCorenlp < Formula
   desc "Java suite of core NLP tools"
   homepage "https://stanfordnlp.github.io/CoreNLP/"
-  url "https://nlp.stanford.edu/software/stanford-corenlp-full-2018-10-05.zip"
-  version "3.9.2"
-  sha256 "833f0f5413a33e7fbc98aeddcb80eb0a55b672f67417b8d956ed9c39abe8d26c"
+  url "https://nlp.stanford.edu/software/stanford-corenlp-4.2.0.zip"
+  version "4.2.0"
```

Comment:
```suggestion
```
We can remove this now that the version can be parsed from the URL.

Reply:
@carlocab Good to know! Thanks.